landing-pages/site/content/en/survey/_index.html

```diff
@@ -1,23 +1,18 @@
 ---
-title: "Airflow Survey 2023"
-linkTitle: "Airflow Survey 2023"
+title: "Airflow Survey 2024"
+linkTitle: "Airflow Survey 2024"
 menu:
   main:
     weight: 31
 ---
 
 <div>
-  <img src="Astronomer - Airflow Survey 2023 - Results - Demographics - Landscape@2x.png"/>
-  <img src="Astronomer - Airflow Survey 2023 - Results - Community and Contribution - Landscape@2x.png"/>
-  <img src="Airflow Survey 2023 Results - Airflow Usage - Page 1 - Revised - Landscape@2x.png"/>
-  <img src="Astronomer - Airflow Survey 2023 - Results - Airflow Usage - Page 2 - Landscape@2x.png"/>
-  <img src="Airflow Survey 2023 Results - Airflow Usage - Page 3 - Revised - Landscape@2x.png"/>
-  <img class="bottom-image" src="Astronomer - Airflow Survey 2023 - Results - Future - Landscape@2x.png"/>
+  <img src="Infographic-Proof 4.png" alt="Airflow Survey 2024 Infographic" />
```

Comment:
```suggestion
  <img src="Airflow Survey 2024 Results.png" alt="Airflow Survey 2024 Infographic" />
```

Lets rename the file to "Airflow Survey 2024 Results.png", it will help with the SEO

Reply:
Changed it.